sui/src/bin/validator.rs

```diff
@@ -51,8 +51,8 @@ async fn main() -> Result<(), anyhow::Error> {
         json_log_output: std::env::var("SUI_JSON_SPAN_LOGS").is_ok(),
         ..Default::default()
     };
-    #[allow(unused)]
-    let guard = telemetry_subscribers::init(config);
+
+    let _guard = telemetry_subscribers::init(config);
```

Comment:
Sorry please don't change this.  Please add a comment to not change it to `_guard` because it causes logging and telemetry subscribers to be dropped right away, thus no logging/tracing.

Reply:
@velvia  I believe `_` will instantly drop but `_sth` will not

Reply:
No we had problems with this in the past, would prefer if you leave it alone

Reply:
I'm pretty sure that `_sth` is equivalent to `sth` with `#[allow(unused)]`, but if you have concerns I'm fine with keeping it in the old way and look into the problems later